tests/unit/ssg-module/test_oval_object_model/test_oval_document.py

```diff
@@ -0,0 +1,51 @@
+import pytest
+
+from test_load_and_store import _load_oval_document, OVAL_DOCUMENT_PATH
```

Comment:
A small nit pick: I think that the tests are somehow exceptional and it actually could be more readable if the path is redefined here in this file instead of importing the constant from other unit test file.